detectable/src/main/java/com/synopsys/integration/detectable/detectables/pnpm/lockfile/process/PnpmLockYamlParserv6.java

```diff
@@ -0,0 +1,94 @@
+package com.synopsys.integration.detectable.detectables.pnpm.lockfile.process;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.collections4.Predicate;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.Nullable;
+
+import com.synopsys.integration.detectable.detectable.codelocation.CodeLocation;
+import com.synopsys.integration.detectable.detectables.pnpm.lockfile.model.PnpmLockYamlv6;
+import com.synopsys.integration.detectable.detectables.pnpm.lockfile.model.PnpmProjectPackagev6;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.util.NameVersion;
+
+public class PnpmLockYamlParserv6 {
+    private static final Predicate<String> isNodeRoot = "."::equals;
```

Comment:
Can you share link to sample v5 vs v6 to visualize the difference?

Reply:
I'll make sure to go over this with the team when we circle next week.